src/helpers/strings-en.ts

```diff
@@ -25,6 +25,7 @@
     "connect-ehr-text-connected": "Connect another provider to share more data or manage your EHR connections.",
     "connect-ehr-not-enabled": "EHR linking is not enabled for this project.",
     "search-for-provider": "Search for Provider or Health Plan",
+    "request-add-provider": "Request to add provider or health plan",
```

Comment:
We should likely make this title dynamic based on whatever categories are configured for the provider search - e.g. "Request to add health plan" if it's just health plan, "Request to add provider" if it's just provider.  I think there's an example of us constructing a string like that in the ExternalAccountsPreview component

Reply:
@chrisnowak I used a separator vs "or".
I'm concerned concatenating the strings together vs translating the 3 potential values will create an output that doesn't align with how it would be naturally spoken
Do you have any thoughts with keeping it as is with the "/" or using "or" making 3 dedicated translations for

Request to add Providers
Request to add Health Plans
Request to add Providers or Health Plans
Not sure if this has come up before

Reply:
I think "/" is fine, "Request to add Providers / Health Plans" seems fine to me